From c8c783a681a0063b245a173759a7aca1ac9452ab Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 15:21:37 -0400
Subject: [infiniband] ocrdma: Add dependency on INET

Message-id: <39eec0ba87f7d6309674f2cde07c2fd040e3cc56.1411571481.git.dledford@redhat.com>
Patchwork-id: 96610
O-Subject: [Patch RHEL7 02/48] RDMA/ocrdma: Add dependency on INET
Bugzilla: 1081067 1088577
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Roland Dreier <roland@purestorage.com>

Now that ocrdma supports IP-based addressing, we need to depend on
INET, since ocrdma registers itself for net device events.

Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 9392fa06411cf93885c4cafc8058085d98f52fec)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/ocrdma/Kconfig b/drivers/infiniband/hw/ocrdma/Kconfig
index b5b6056..c0cddc0 100644
--- a/drivers/infiniband/hw/ocrdma/Kconfig
+++ b/drivers/infiniband/hw/ocrdma/Kconfig
@@ -1,6 +1,6 @@
 config INFINIBAND_OCRDMA
  tristate "Emulex One Connect HCA support"
- depends on ETHERNET && NETDEVICES && PCI && (IPV6 || IPV6=n)
+ depends on ETHERNET && NETDEVICES && PCI && INET && (IPV6 || IPV6=n)
  select NET_VENDOR_EMULEX
  select BE2NET
  ---help---
-- 
1.7.1